### PR TITLE
Add To Do App code

### PR DESCRIPTION
**Pull request by adelasimnova**

### CODE REVIEW

File: src/card/Card.css [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    max-width: 810px;
+    display: flex; 
+    align-items: center;
+    justify-content: space-between;
+}
+
+.delete-button {
+    width: 50px;
+    height: 50px;
+    border: none;
```
Comment (adelasimnova) [inline src/card/Card.css:16]: Test comment

File: .eslintrc.cjs [hunk 2]
```diff
@@ -17,5 +17,7 @@ module.exports = {
       'warn',
       { allowConstantExport: true },
     ],
+    'react/prop-types': 0,
```
Comment (KatarinaSimnova) [inline .eslintrc.cjs:20]: test comment

Comment (adelasimnova) [reply .eslintrc.cjs:20]: Test answer

**Comment by KatarinaSimnova** [comment]
Remove "public" folder.

File: package.json
Comment (KatarinaSimnova) [inline package.json:1]: Change "name" property. It should be "todo-app".
Also "version" should be 1.0.0.

**Comment by KatarinaSimnova** [comment]
Don't forget to add project description to README.

**Comment by KatarinaSimnova** [comment]
Remove src/assets folder.

File: src/to-do-list/ToDoList.jsx [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+import 'boxicons';
+
+export function ToDoList() {
+  const [todos, setTodos] = React.useState([]);
+  const [loading, setLoading] = React.useState(false);
+  const [error, setError] = React.useState(null);
+
+  React.useEffect(() => {
+    setLoading(true);
+    getTodos().then((response) => {
```
Comment (KatarinaSimnova) [inline src/to-do-list/ToDoList.jsx:23]: Handle error in catch block.

File: src/to-do-list/ToDoList.jsx [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+      setLoading(false);
+      return;
+    }
+    if (toDoData.title.length >= 30) {
+      setError('Text is too long!');
+      setLoading(false);
+      return;
+    }
+    createTodo(toDoData).then(() => {
+      getTodos().then((response) => {
```
Comment (KatarinaSimnova) [inline src/to-do-list/ToDoList.jsx]: Handle error in catch block.

File: src/to-do-list/ToDoList.jsx [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+        setTodos(response.data.todos);
+        setLoading(false);
+      });
+    });
+  }
+
+  function deleteToDo(todoId) {
+    setLoading(true);
+    deleteTodo(todoId).then(() => {
```
Comment (KatarinaSimnova) [inline src/to-do-list/ToDoList.jsx:57]: Handle error in catch block.

File: src/to-do-list/ToDoList.jsx [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+    updateTodo(todoId, toDoData).then(() => {
+      getTodos().then((response) => {
+        setTodos(response.data.todos);
+        setLoading(false);
+      });
+    });
+  }
+
+  return (
+    <div className="card-list">
```
Comment (KatarinaSimnova) [inline src/to-do-list/ToDoList.jsx]: Rename "card-list" class to something more suitable.

File: src/to-do-form/ToDoForm.css [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+    border: none;
+    border-radius: 5px;
+    font-weight: lighter;
+    font-size: xx-large;
+    color: white;
+    background-color: rgb(29, 29, 195);
+    cursor: pointer;
+}
+
+/* @media (max-width: 376px) {
```
Comment (KatarinaSimnova) [inline src/to-do-form/ToDoForm.css]: Remove comment.

File: src/to-do-form/ToDoForm.css [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+    font-size: larger;
+}
+
+.to-do-button {
+    width: 50px;
+    height: 50px;
+    border: none;
+    border-radius: 5px;
+    font-weight: lighter;
+    font-size: xx-large;
```
Comment (KatarinaSimnova) [inline src/to-do-form/ToDoForm.css]: Let's define font-size in pixels.

File: src/main.jsx [hunk 9]
```diff
@@ -1,10 +1,10 @@
 import React from 'react'
 import ReactDOM from 'react-dom/client'
-import App from './App.jsx'
+import { ToDoList } from './to-do-list/ToDoList';
 import './index.css'
 
 ReactDOM.createRoot(document.getElementById('root')).render(
   <React.StrictMode>
```
Comment (KatarinaSimnova) [inline src/main.jsx]: Remove "React.StrictMode"

**Comment by KatarinaSimnova** [comment]
Don't forget to add project description to README.

**Review by KatarinaSimnova — APPROVED** [review]

**Pull request merged**